packages/cli/docs-resolver/src/DocsDefinitionResolver.ts

```diff
@@ -506,7 +529,7 @@ export class DocsDefinitionResolver {
             orphaned: landingPageConfig.orphaned,
             pageId,
             authed: undefined,
-            noindex: undefined,
+            noindex: landingPageConfig.noindex ?? this.markdownFilesToNoIndex.get(landingPageConfig.absolutePath),
```

Comment:
thous this be `landingPageConfig.noindex || this.markdownFilesToNoIndex.get(landingPageConfig.absolutePath)`